4-generalization/labs/05-peoples-park/lab.qmd

```diff
@@ -35,57 +35,76 @@ In answering the following questions, it will be helpful to consult
 
-<!-- #### Question 1 -->
+#### Question 1
 
-<!-- Print the first few rows with the columns that correspond to the responses to survey questions 1, 7, and 8. Note: we have changed the data back from all numerical data, as suggested by lab question 8, to a mix of numerical and categorical data. Please comment on whether your encoding of the data from Q7 on the questionnaire matches the encoding in `ppk`. -->
+Print the first few rows with the columns that correspond to the responses to survey questions 1, 7, and 8 (Note: we have changed the data back from all numerical data, as suggested by part 1 question 8, to a mix of numerical and categorical data). Please comment on whether your encoding of the data from Q7 on the questionnaire matches the encoding in `ppk`.
 
-<!-- #### Question 2 -->
+#### Question 2
 
-<!-- Return to your sketches from question 9 here in the lab. Create those visualizations (or more appropriate analogues) using the questionnaire data. For each, add a title and axis labels to make it clear what they are showing, and describe the distribution in words. If your visualization is of ordinal data, the bars should be ordered accordingly. For part a here, you're welcome to select just three of the priorities to visualize. -->
+Return to your sketches from lab part 1, question 9. Create those visualizations (or more appropriate analogues) using the questionnaire data. For each, add a title and axis labels to make it clear what they are showing, and describe the distribution in words. If your visualization is of ordinal data, the bars should be ordered accordingly. For part a here, you're welcome to select just one of the priorities to visualize.
 
-<!-- a. Question 9 -->
-<!-- b. Question 10 -->
-<!-- c. Question 18 and 21 (showing the change from before and after the information in one plot) -->
+a. Question 9
+b. Question 10
+c. Question 18 and 21 (showing the change from before and after the information in one plot)
 
-<!-- #### Question 3 -->
+#### Question 3
 
-<!-- Create a new column called `support_before` that takes the response data from question 18 and returns `TRUE` for answers of "Very strongly support", "Strongly support", and "Somewhat support" and FALSE otherwise. What proportion of the survey participants in each class (freshman, sophomore, etc) supported the People's Park Project before being presented with the information on the bottom of page 14? -->
+Create a new column called `support_before` that takes the response data from question 18 and returns `TRUE` for answers of "Very strongly support", "Strongly support", and "Somewhat support" and FALSE otherwise. What proportion of the survey participants in each class (freshman, sophomore, etc) supported the People's Park Project before being presented with the information on the bottom of page 14?
 
-<!-- #### Question 4 -->
 
-<!-- What is the mean and median rating of the condition of People's Park (question 15 on the survey)? -->
+#### Question 4
 
```

Comment:
@jeremy-sanchez this is a new question, added to address the `drop_na()` issue that can effect many of these answers. this isn't ideal - introducing a function mid-lab - but i can't think of a better approach and we do do that with the lubridate function in the flights lab.